src/lib.rs

```diff
@@ -11,13 +11,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! A failpoints implementation for rust.
+//! A fail points implementation for rust.
+//!
+//! Fail points are code points that used to inject error by user at runtime.
+//! This crate is inspired by FreeBSD's [failpoints](https://freebsd.org/cgi/man.cgi?query=fail).
+//! Unlike FreeBSD's implementation, this crate only support configuration from
+//! environment variables and API.
```

Comment:
-> this crate only supports the configuration from environment variables and HTTP API.

Reply:
Environment variable, API and HTTP API are planned to be supported. But only first two methods are implemented.